index.js

```diff
@@ -30,7 +30,10 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) { 
+  if (movies.length === 0) throw "There are no movies"
```

Comment:
Try to follow the rule that `if` and `else` statements should have code blocks even for single statements.